src/index.js

```diff
@@ -35,13 +30,10 @@ export default function supernova(env) {
       const constraints = useConstraints();
       const selectionsAPI = useSelections();
       const theme = useTheme();
-
       const [pageInfo, setPageInfo] = useState({ top: 0, height: 100 });
-      const [muiParameters] = useState(muiSetup(__OPIONAL_THEME_DEPS__));
-      const [tableData] = usePromise(() => manageData(model, layout, pageInfo), [layout, pageInfo]);
```

Comment:
in this case, at first, tableData is empty, then it got value, which results in that component() run twice.